src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/templates/VersionPage/MetadataTabTests.kt

```diff
@@ -42,6 +42,7 @@ class MetadataTabTests: BaseVersionPageTests()
         assertThat(content.select("#git-hr").count()).isEqualTo(0)
         assertThat(content.select("#git-branch-row").count()).isEqualTo(0)
         assertThat(content.select("#git-commit-row").count()).isEqualTo(0)
+        assertThat(content.select("#db-instance-row").count()).isEqualTo(0)
```

Comment:
Should this be `.db-instance-row`?

This test probably needs renaming as testing for instances too.